x-pack/plugins/reporting/common/types.ts

```diff
@@ -50,7 +50,6 @@ export interface TaskRunResult {
   size: number;
   csv_contains_formulas?: boolean;
   max_size_reached?: boolean;
-  needs_sorting?: boolean;
```

Comment:
This line of code was accidentally added in an earlier PR https://github.com/elastic/kibana/pull/88303